rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -859,7 +859,7 @@ private boolean isNumberOfNodesLimited() {
      *
      * @return total number of alive nodes
      */
-    private int getTotalAliveNodesNumber() {
+    public int getTotalAliveNodesNumber() {
```

Comment:
Would it make sense to add this method to ResourceManager interface ?

Reply:
I will use listAliveNodeUrls() that already exists in the ResourceManager interface